{"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"attribute","text":"@objc"},{"kind":"text","text":" "},{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"message"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"}],"languages":["swift"],"platforms":["macOS"]}]}],"schemaVersion":{"major":0,"minor":3,"patch":0},"sections":[],"variants":[{"paths":["\/documentation\/snowplowtracker\/trackererror\/message"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"url":"doc:\/\/SnowplowTracker\/documentation\/SnowplowTracker\/TrackerError\/message","interfaceLanguage":"swift"},"abstract":[{"type":"text","text":"Message of the error."}],"kind":"symbol","metadata":{"fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"message"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"}],"title":"message","roleHeading":"Instance Property","role":"symbol","symbolKind":"property","externalID":"c:@M@SnowplowTracker@objc(cs)SPTrackerError(py)message","modules":[{"name":"SnowplowTracker"}]},"hierarchy":{"paths":[["doc:\/\/SnowplowTracker\/documentation\/SnowplowTracker","doc:\/\/SnowplowTracker\/documentation\/SnowplowTracker\/TrackerError"]]},"references":{"doc://SnowplowTracker/documentation/SnowplowTracker/TrackerError":{"role":"symbol","title":"TrackerError","fragments":[{"kind":"keyword","text":"class"},{"kind":"text","text":" "},{"kind":"identifier","text":"TrackerError"}],"abstract":[{"type":"text","text":"Tracker error event used in diagnostic autotracking."}],"identifier":"doc:\/\/SnowplowTracker\/documentation\/SnowplowTracker\/TrackerError","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"TrackerError"}],"url":"\/documentation\/snowplowtracker\/trackererror"},"doc://SnowplowTracker/documentation/SnowplowTracker/TrackerError/message":{"role":"symbol","title":"message","fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"message"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"}],"abstract":[{"type":"text","text":"Message of the error."}],"identifier":"doc:\/\/SnowplowTracker\/documentation\/SnowplowTracker\/TrackerError\/message","kind":"symbol","type":"topic","url":"\/documentation\/snowplowtracker\/trackererror\/message"},"doc://SnowplowTracker/documentation/SnowplowTracker":{"role":"collection","title":"SnowplowTracker","abstract":[],"identifier":"doc:\/\/SnowplowTracker\/documentation\/SnowplowTracker","kind":"symbol","type":"topic","url":"\/documentation\/snowplowtracker"}}}